WebCryptoAPI/sign_verify/eddsa_vectors.js

```diff
@@ -0,0 +1,58 @@
+// eddsa_vectors.js
+
+// Data for testing EdDSA with every curve currently in the WebCryptoAPI recommendation.
```

Comment:
```suggestion
// Data for testing Ed25519 and Ed448.
```
(since they aren't a W3C Recommendation yet)